### DISCUSSION TITLE
Have an option for the audio device button on the top of the flyout to behave like the stock volume menu

### DISCUSSION THREAD

**Discussion by hifinerd** [File-New-Project/EarTrumpet#1026] [Ideas]
I really like this app so far, but my only gripe with it is the fact that the audio device button (the one on the top that shows the name of your audio device) extends the menu to show what those devices are playing. Is it possible to have this button act like the stock flyout, where it just shows a menu of what audio device you want to use?

**Comment by riverar** [comment]
@hifinerd Just a heads up, you can switch audio devices by right-clicking the EarTrumpet icon.
